guides/other/withdraw-agEUR-bridge.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Exchange your lz-agEUR for agEUR
```

Comment:
Let's rather add this page in the bridge guide -> so all is in one page and not multiple pages.

We also need to add details on the json-RPC error